projects/plugins/jetpack/modules/widgets/eu-cookie-law/form.php

```diff
@@ -1,7 +1,13 @@
 <?php
+/**
+ * EU Cookie Law Widget form.
+ *
+ * @package automattic/jetpack
+ */
 
 use Automattic\Jetpack\Redirect;
 
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
I suppose if you really wanted to you could do something like https://github.com/Automattic/jetpack/blob/6d1bce942323a8735acad9bffa4cefb626e298d4/projects/plugins/beta/src/admin/exception.template.php#L13-L17 to define the two variables that are supposed to be supplied by the outer scope.

But if you don't want to, that's ok with me.

Reply:
Thanks for sharing that example 👍  I think I'll leave as-is for now to better match other files already using the ignore.